app/components/finance/statements/output_payments/ect_mentor.html.erb

```diff
@@ -0,0 +1,87 @@
+<div class="finance-panel finance-panel__output-payments">
+  <%= govuk_table classes: ["output-payments"] do |table| %>
+    <% table.with_caption(size: 'm', text: "Early career teacher (ECT) output payments") %>
+
+    <% table.with_head do |head| %>
+      <% head.with_row do |row| %>
+        <% row.with_cell(header: true, text: "Outputs") %>
+
+        <% ect_calculator.band_letters.each do |letter| %>
+          <% row.with_cell(header: true, text: "Band #{letter.upcase}", numeric: true) %>
+        <% end %>
+
+        <% row.with_cell(header: true, text: "Payments", numeric: true) %>
+      <% end %>
+    <% end %>
+
+    <% table.with_body do |body| %>
```

Comment:
maybe this can be extracted into it's own component as its duplicated across the old world AND new world?

Reply:
the output payments for new way is inside one "finance-panel" block, thought it would be better together.  that way we have one component for old and one for new.